lib/service/service.go

```diff
@@ -66,21 +66,46 @@ var log = logrus.WithFields(logrus.Fields{
 })
 
 const (
-	// ProxyReverseTunnelServerEvent is generated supervisor when proxy
-	// has initialized reverse tunnel server
-	ProxyReverseTunnelServerEvent = "ProxyReverseTunnelServer"
-	// ProxyWebServerEvent is generated supervisor when proxy
-	// has initialized web tunnel server
-	ProxyWebServerEvent = "ProxyWebServer"
-	// ProxyIdentityEvent is generated by supervisor when proxy's identity has been initialized
+	// AuthIdentityEvent is generated when the Auth Servers identity has been
+	// initialized in the backend.
+	AuthIdentityEvent = "AuthIdentity"
+
+	// ProxyIdentityEvent is generated by the supervisor when the proxy's
+	// identity has been registered with the Auth Server.
 	ProxyIdentityEvent = "ProxyIdentity"
-	// SSHIdentityEvent is generated when node's identity has been received
+
+	// SSHIdentityEvent is generated when node's identity has been registered
+	// with the Auth Server.
 	SSHIdentityEvent = "SSHIdentity"
-	// TeleportExitEvent is generated when someone is askign Teleport Process to close
-	// all listening sockets and exit
+
+	// DELETE IN: 2.6.0
+	// AuthServerSSHReady is generated when the Auth Server has initialized the
+	// HTTP-over-SSH endpoint and is ready to start accepting connections.
+	AuthServerSSHReady = "AuthServerReady"
+
+	// AuthServerTLSReady is generated when the Auth Server has initialized the
+	// TLS Mutual Auth endpoint and is ready to start accepting connections.
+	AuthServerTLSReady = "AuthServerReady"
```

Comment:
the name does not match the value, is this on purpose?

Reply:
Typo.